src/components/Common/MainMenu.vue

```diff
@@ -147,20 +147,10 @@ export default {
     async doLogout() {
       try {
         await this.$store.direct.dispatch.auth.doLogout()
-        this.$router.push({ name: 'Login' })
```

Comment:
Why this is deleted?

Reply:
I use a `finally` statement because we want to redirect users to the login page either logout succeed or failed

Reply:
oh, right!